docs/appendix.rst

```diff
@@ -524,76 +509,190 @@ To fix this, install the Python "wheel" package::
 
     pip install wheel
 
-ConnectionError raised with [SSL: UNSUPPORTED_PROTOCOL]
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. index:: pair; troubleshooting: OpenSSL
+.. index:: pair; installation fail: OpenSSL
+
+NotOpenSSLWarning: urllib3 v2.0 only supports OpenSSL 1.1.1+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. index:: pair; installation fail: NotOpenSSLWarning
+.. index:: pair; troubleshooting: NotOpenSSLWarning
+
+This issue is probably caused by the dependent Python package urllib3 update to
+version 2.0 in pywbem version 1.7.0. In this case probably the local OS
+environment includes a version of OpenSSL less than 2.0 or another SSL
+implementation.
+
+See :ref:`ConnectionError raised with SSL UNSUPPORTED_PROTOCOL` for
+more information.
+
+
+.. _ConnectionError raised with SSL UNSUPPORTED_PROTOCOL:
+
+ConnectionError raised with SSL UNSUPPORTED_PROTOCOL
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 .. index:: pair; troubleshooting: OpenSSL
 .. index:: pair; SSL: OpenSSL
+.. index:: single LibreSSL
+.. index:: single: TLS
 .. index:: pair; UNSUPPORTED_PROTOCOL: OpenSSL
 
 On newer versions of the operating system running the pywbem client,
-communication with the WBEM server may fail with::
+communication with the WBEM server may fail with an exception similar to::
 
     pywbem.exceptions.ConnectionError: SSL error <class 'ssl.SSLError'>:
       [SSL: UNSUPPORTED_PROTOCOL] unsupported protocol (_ssl.c:1056)
 
-In pywbem version 1.7.0 the urllib3 configuration version limit was modified to allow
-urllib3 versions >= 2.0.  These versions of urllib3 limit the minimum TLS
-version to 1.2 (i.e OpenSSL version >= 1.1.1). If the version of OpenSSL is
-less than 1.1.1, this SSLError will occur with the initial request to the
-WBEM Server. Urllib3 version <= 2.0 also limits the SSL library implementations
-to just OpenSSL and possibly libreSSL.
+This error means that the WBEM server side SSL implementation does not yet support
+TLS 1.2 or higher or that the SSL library (OpenSSL or LibreSSL) used
+by pywbem does not support TLS 1.2 and the other side requires TLS >= 1.2.
+
+See the Python document
+`Python PEP 644 – Require OpenSSL 1.1.1 or newer <https://peps.python.org/pep-0644/>`_
+for more information on Python and OpenSSL version 1.1.1.
+
+In 'pywbem' package version 1.7.0 the urllib3 package configuration version
+limit was modified to allow urllib3 package versions >= 2.0.  These new
+versions of the 'urllib3' package include backward incompatible changes
+including:
```

Comment:
In pywbem 1.6.0, urllib3 was not pinned, which means that urllib3 2.0 can be used with it. So this is not something that pywbem 1.7.0 introduced, it is rather tied to the release of urllib3 2.0.